ice/di/access.zep

```diff
@@ -25,15 +25,13 @@ class Access
     {
         var di, service;
 
-        if typeof this->di != "object" || typeof this->di == "object" && !(this->di instanceof Di) {
```

Comment:
I remember it wasn't possible to check `instanceof` on non-object on zephir before

Reply:
alright, updated